packages/wallet-service/serverless.yml

```diff
@@ -242,6 +242,11 @@ functions:
           arn:aws:lambda:eu-central-1:${self:provider.environment.ACCOUNT_ID}:function:hathor-explorer-service-${self:custom.explorerServiceStage}-create_or_update_dag_metadata
   loadWalletAsync:
     handler: src/api/wallet.loadWallet
+    # This lambda is currently running out of memory when wallets with a big
+    # number of transactions (> 300k) is being loaded. I chose 1024 because the
+    # average memory usage of the largest test wallet we have is 800mb.
+    memorySize: 1024
+    timeout: 600 # 10 minutes should be enough for most wallets
```

Comment:
[suggestion] Can we use a cache to store all these data outside the lambda and yet make it available extremely fast? Maybe another product from AWS?

Reply:
The wallet-service is the cache, this method creates it

This lambda receives the wallet's `xpubkey` and uses it to relate all addresses and their transactions and utxos (which are already processed) to the newly created wallet

This process is what is using more memory than it should 

Without the `xpubkey` we can never know which address belongs to which wallet